client/components/utils/title-size-plugin.js

```diff
@@ -0,0 +1,31 @@
+import {Plugin} from "prosemirror-state";
+import {Snackbar} from '../../snackbar';
+
+export const titleSizePlugin = (max, message) => {
+  let lastOccurrenceAt = 0;
+  return new Plugin({
+    filterTransaction(tr, state) {
+      // Check if the transaction title length is greater than the limit
+      if (tr.doc.content.firstChild.content.size > max) {
```

Comment:
Could you maybe move this function somewhere and reuse it: https://github.com/peer/doc/blob/devel/server/documents/content.js#L15